src/components/filter/expert/expertFilterConstants.ts

```diff
@@ -1041,6 +1041,16 @@ export const FIELDS_OPTIONS = {
         label: 'tieLineId',
         dataType: DataType.STRING,
     },
+    SHORT_CIRCUIT_CURRENT_LOW_LIMIT: {
+        name: FieldType.SHORT_CIRCUIT_CURRENT_LOW_LIMIT,
+        label: 'shortCircuitLowCurrentLimit',
+        dataType: DataType.NUMBER,
+    },
+    SHORT_CIRCUIT_CURRENT_HIGH_LIMIT: {
+        name: FieldType.SHORT_CIRCUIT_CURRENT_HIGH_LIMIT,
+        label: 'shortCircuitHighCurrentLimit',
+        dataType: DataType.NUMBER,
+    },
```

Comment:
In the previous enum value using `DataType.NUMBER`, like `DC_NOMINAL_VOLTAGE`, there was `inputType: 'number',` as well. I don't know the reason but maybe it is important...?